tests/unittests/Foundation/NSJSONSerializationTests.m

```diff
@@ -124,13 +124,13 @@ void VerifyDataWithJSONObjectThrows(id testObject) {
     id testObject6 = @{ @"foo" : @"1", @"bar" : @"2" };
     id testObject7 = @{ @"foo" : @{ @"bar" : @"1" }, @"foo2" : @{ @"bar2" : @"2" } };
     id testObject8 = [NSUUID UUID];
-    id testObject9 = @{ @"key" : @"value", @"nullkey" : [NSNull null], @"array" : @[ @{ @"subnullkey" : [NSNull null] } ] };
```

Comment:
Alternatively, we could reconstruct these test cases to test single-element dictionaries. We can't lose fidelity, though, so we need to test everything we would have otherwise tested for a multi-member dictionary.

---
In reply to: [80983725](https://github.com/Microsoft/WinObjC/pull/1043#discussion_r80983725) [](ancestors = 80983725,80978644,80960041,80824620)